src/utils/common.ts

```diff
@@ -256,18 +256,21 @@ export async function estimateTotalGasRequiredByUnsignedTransaction(
   options: Partial<{
     gasPrice: BigNumberish;
     gasUnits: BigNumberish;
+    baseFeeMultiplier: number;
```

Comment:
@pxrl I added in an optional multiplier arg here. I don't think we want to use this anywhere in the SDK since all calls to this function only care about the `gasCost`. In fact, we might want to change this function soon to only return the `gasCost` and simplify the interface